---
permalink: /
title: About me
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
I am a Machine Learning Research Scientist at [Natera](https://www.natera.com/) in the Therapeutics AI team, where I focus on developing genomic large language models (LLMs) for clinical applications and designing deep learning methods for neoantigen prediction in cancer immunotherapy.
Previously, I obtained my PhD at the [Data Science and Mining Team](http://www.lix.polytechnique.fr/dascim/) of the Computer Science Laboratory at [Ã‰cole Polytechnique](https://www.polytechnique.edu/en), under the supervision of Prof. [Michalis Vazirgiannis](http://www.lix.polytechnique.fr/~mvazirg/). 
In 2024, I completed an internship at [InstaDeep](https://www.instadeep.com/) working on graph neural networks and single cell foundation models.
In 2022, I completed an internship at [Flatiron Institute](https://www.simonsfoundation.org/flatiron/) of Simons Foundation in New York, where I worked on the application of graph neural networks for cancer gene prediction under the guidance of Dr. [Zijun Frank Zhang](https://scholar.google.com/citations?user=YDl3mfgAAAAJ&hl=en).
Prior to this, I graduated from the School of Electrical and Computer Engineering at the [National Technical University of Athens (NTUA)](https://www.ece.ntua.gr/en). During my undergraduate studies, I worked as a Machine Learning Researcher in [Laboratory of Algebraic and
Geometric Algorithms](http://erga.di.uoa.gr/) at [National and Kapodistrian University of Athens](https://www.di.uoa.gr/en) under the supervision of  Prof. [Ioannis Emiris](https://scholar.google.gr/citations?user=ZK6y-cIAAAAJ&hl=en). I also successfully completed Google Summer of Code 2019.

My research interests lie at the intersection of machine learning, graph representation learning, and foundation models for biology. I am particularly interested in developing methods that combine structured, multimodal, and generative approaches to better model complex biological systems. Beyond biology, I am also exploring how these advances can contribute to broader areas such as automated machine learning.


<h2> News </h2>
<ul>
<li> <strong> 2025-09-19 </strong> Our paper <strong><a href=https://arxiv.org/abs/2505.11194>Prot2Text-V2: Protein Function Prediction with Multimodal Contrastive Alignment
</a> </strong> has been accepted at <strong>NeurIPS 2025</strong>!
<li> <strong> 2025-07-15 </strong> I joined <strong>Natera</strong> as a <strong>ML Research Scientist</strong>!
<li> <strong> 2025-04-10 </strong> I successfully defended my PhD thesis titled <strong>Advancements in Graph Representation Learning and Applications in Computational Biology</strong> <a href="https://theses.hal.science/tel-05118611/">https://theses.hal.science/tel-05118611/</a>. 
<li> <strong> 2024-06-01 </strong> I started a <strong>research internship</strong> at <strong>InstaDeep</strong> working on GNNs and LLMs for genomics. 
<li> <strong> 2023-01-15 </strong> Happy to serve as a mentor at <strong><a href="https://www.logml.ai/"> LOGML Summer School 2024 </a> </strong></li>
<li> <strong> 2023-12-09</strong> Our paper <strong><a href="https://arxiv.org/abs/2307.14367">Prot2Text: Multimodal Protein's Function Generation with GNNs and Transformers</a> </strong> has been accepted at <strong>AAAI 2024!</strong> </li>
<li> <strong> 2023-10-27 </strong> Our paper <strong><a href="https://arxiv.org/abs/2307.14367">Prot2Text: Multimodal Protein's Function Generation with GNNs and Transformers</a> </strong> has been accepted as <strong>spotlight</strong> at DGM4H and AI4Science at NeurIPS 2023!
<li> <strong> 2023-10-17 </strong> Our paper <strong><a href="https://arxiv.org/pdf/2301.08831.pdf">Explainable Multilayer Graph Neural Network for Cancer Gene Prediction</a> </strong> has been accepted at <strong>Bioinformatics</strong>!
  <li> <strong> 2023-06-29 </strong> Our paper "Supervised Attention Using Homophily in Graph Neural Networks" has been accepted at <strong>ICANN 2023</strong>!
  <li> <strong> 2023-02-16 </strong> Our paper <strong><a href="https://arxiv.org/pdf/2302.05894.pdf">Neural Architecture Search with Multimodal Fusion Methods for Diagnosing Demenntia</a> </strong> has been accepted at <strong>ICASSP 2023</strong>!
  <li> New paper preprint <strong><a href="https://arxiv.org/pdf/2301.08831.pdf">Explainable Multilayer Graph Neural Network for Cancer Gene Prediction</a> </strong>  </li>
  <li> <strong> 2023-01-23 </strong> Our paper <strong><a href="https://michailchatzianastasis.github.io/publication/wlhn">Weisfeiler and Leman go Hyperbolic: Learning Distance Preserving Node Representations</a> </strong> has been accepted at <strong>AISTATS 2023</strong>!
  <li> <strong> 2022-11-21 </strong> Pleased to announce that our paper <strong><a href="https://michailchatzianastasis.github.io/publication/goat">Graph Ordering Attention Networks</a> </strong> has been accepted at <strong>AAAI 2023</strong>!
  <li>
  <strong>2022-11-04</strong> New paper Preprint: <strong><a href="https://michailchatzianastasis.github.io/publication/wlhn">Weisfeiler and Leman go Hyperbolic: Learning Distance Preserving Node
Representations</a> </strong> 
  </li>
  <li> <strong> 2022-06-28 </strong> We presented our paper <strong><a href="https://michailchatzianastasis.github.io/publication/goat">Graph Ordering Attention Networks</a> </strong> in the <a href="https://hannes-stark.com/logag-reading-group">LoGaG: Learning on Graphs and Geometry Reading Group. </a> <strong> <a href="https://www.youtube.com/watch?v=9CJs5TL-CSs">[Video]</a> </strong>

  <li> <strong> 2022-06-13</strong> Pleased to anounce that our paper <strong> <a href="https://www.biorxiv.org/content/10.1101/2022.06.22.497214v1"> Mass Enhanced Node Embeddings for Drug Repurposing 
  </a> </strong> with Giannis Nikolentzos and Michalis Vazirgiannis, was accepted at ICML Workshop on Computational Biology 2022. 
  <li> <strong>2022-06-01</strong> I will spend this summer at <a href="https://www.simonsfoundation.org/grant/2022-flatiron-machine-learning-x-science-summer-school/"> Flatiron's Machine Learning Summer School </a> in New York, working on graph neural networks for cancer risk prediction.
  <li>We present our paper <strong><a href="https://michailchatzianastasis.github.io/publication/operation_embeddings_for_nas">Graph-based Neural Architecture Search with Operation Embeddings</a> </strong>  at <a href="https://neural-architecture-ppf.github.io/">ICCV 2021 NAS Workshop </a>. 
  <a href="https://arxiv.org/abs/2105.04885">Paper</a>, <a href="https://github.com/MichailChatzianastasis/Graph-based_NAS_with_Operation_Embeddings">Code</a>,  <a href="https://www.youtube.com/watch?v=-rZ4tpNvL6s&t=1s">Video</a>, <a href="https://github.com/MichailChatzianastasis/Graph-based_NAS_with_Operation_Embeddings/blob/master/ Graph_based_neural_architecture_search_with_operation_embeddings_ICCV.pdf">Poster</a> 
  </li>


